I-D_in_xml/draft-ietf-ccamp-optical-impairment-topology-yang.xml

```diff
@@ -300,6 +299,12 @@
 
 ]]></artwork>
   </figure>
+
+  <t>
+   Note: The RFC Editor will replace AAAA above with the number
+   assigned to the RFC once draft-ietf-ccamp-dwdm-if-param-yang will
+   become an RFC.
```

Comment:
I think that in the figure above the following change is also needed

OLD
`
Scope of [I-D.ietf-ccamp-dwdm-if-param-yang]
`
NEW
`
Scope of [RFCAAAA]`

Reply:
Resolved with https://github.com/ietf-ccamp-wg/draft-ietf-ccamp-optical-impairment-topology-yang/pull/152/commits/a41ccfd10d98f4bdd51d067442c8fcd3164f1b92